auth/credentials/filetypes.go

```diff
@@ -41,12 +41,12 @@ func fileCredentials(b []byte, opts *DetectOptions) (*auth.Credentials, error) {
 		if err != nil {
 			return nil, err
 		}
-		tp, err = handleServiceAccount(f, opts)
+		universeDomain = resolveUniverseDomain(opts.UniverseDomain, f.UniverseDomain)
```

Comment:
nit: move this into the function call instead of passing it here. You have all the info you need for it there.

Reply:
The `universeDomain` local var that is assigned here is used on line 111, below. But I could avoid the new param by simply calling `resolveUniverseDomain ` a second time inside `resolveUniverseDomain`.

Reply:
ah this is fine then, did not notice that!

Reply:
Done, thanks.